src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -340,6 +358,7 @@ export const ArtworkAppFragmentContainer = createFragmentContainer(
         is_offerable: isOfferable
         availability
         visibilityLevel
+        isSold
```

Comment:
I think we can safely remove the `isSold` from here, don't think we use it anywhere
```suggestion
```

Reply:
good catch for these leftovers, thank you!